package.json

```diff
@@ -72,7 +73,9 @@
     "generate-share-images": "ts-node-script -r tsconfig-paths/register scripts/generate_share_images/index.ts",
     "generate-daily-alerts": "ts-node-script -r tsconfig-paths/register scripts/alert_emails/generate_daily_alerts.ts",
     "send-emails": "ts-node-script -r tsconfig-paths/register scripts/alert_emails/send_emails.ts",
-    "send-test-email": "ts-node-script -r tsconfig-paths/register scripts/alert_emails/send-test-email.ts"
+    "send-test-email": "ts-node-script -r tsconfig-paths/register scripts/alert_emails/send-test-email.ts",
+    "report-alert-emails": "ts-node-script -r tsconfig-paths/register scripts/alert_emails_report/generate-report.ts",
+    "gs": "ts-node-script -r tsconfig-paths/register scripts/common/google-sheets.ts"
```

Comment:
is this just temporary for testing?

Reply:
Yes, sorry, I forgot to remove it 😅 - it works though!